packages/e2e-test-utils-playwright/tsconfig.json

```diff
@@ -4,6 +4,8 @@
 		"incremental": false,
 		"composite": false,
 		"module": "CommonJS",
+		"moduleResolution": "node16",
+		"skipLibCheck": true,
```

Comment:
Hacky, but successfully ignores TS error flagged in `node_modules/lighthouse/core/index.d.cts`

Reply:
Is it possible to patch the TS errors locally? NVM if it's too much effort though!

Reply:
Do we have patch-package in the repo?
Ideally we don‘t need either though, so would love some insight. Best to remove this locally and see the error message yourself.

Reply:
Resolved now with patch-package